### ISSUE TITLE
v1.0.0 Release 🚀

### ISSUE THREAD

**Issue by fsdiogo** [ipfs-shipyard/ipfs-share-files#11]
Release checklist:
- [x] Add page
- [x] Download page
- [x] Update [share.ipfs.io](https://share.ipfs.io) and [dev.share.ipfs.io](https://dev.share.ipfs.io)
- [x] Don't rely on the public gateway (https://github.com/ipfs-shipyard/ipfs-share-files/issues/43)
- [x] Update copy and assets (https://github.com/ipfs-shipyard/ipfs-share-files/issues/41)
- [x] Add project to Transifex (https://github.com/ipfs-shipyard/ipfs-share-files/issues/33)
- [ ] Fix https://github.com/ipfs/js-ipfs-api/issues/797 (https://github.com/ipfs-shipyard/ipfs-share-files/issues/22)
- [ ] Post a link in [this thread](https://discuss.ipfs.io/t/is-there-any-other-methods-to-upload-a-file-files-to-ipfs-through-a-web-browser-beside-using-api-port-5001/3143/5)

**Comment by lidel** [comment]
Let's add translations to the list  – I can help with that :)

For "WebUI" integration take a look at what @olizilla did with IPLD Explorer:

- https://github.com/ipfs-shipyard/ipld-explorer 
  standalone app that depends on `ipld-explorer-components`
- https://github.com/ipfs-shipyard/ipld-explorer-components
  a dependency used by both `ipld-explorer` and `ipfs-webui`

@olizilla Please confirm if creating `ipfs-share-files-components` and following Explorer's conventions is the right way to do this?

**Comment by hacdias** [comment]
Hmm, I thought we were just giving links for `share.ipfs.io/#/add/hash/hash2` and not really embedding this into WebUI.

**Comment by olizilla** [comment]
The dream is to have a bunch of separate apps that link out to each other. We're going to try it out with the share app to see what the UX is like.

For context, I'm not super happy with `ipld-explorer-components` but it's the least worst solution for now, to get webui to a state we can release it. Once WebUI is out there, then we'll be focusing on building out the reusable component library that @hacdias has always dreamed of. 

Composite components (things more complex than "Button", like IPLD Explorer) are tricky, as, while there are a many ways to share css styles from reusable components, the path for publishing components with i18n locales is much less clear, but we'll find a way.

**Comment by olizilla** [comment]
We need a fix for https://github.com/ipfs/js-ipfs-api/issues/797#issuecomment-401469158 before we can release this

**Comment by jessicaschilling** [comment]
Closing - stale, and superseded by this week's hackathon activity.
